<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mini-Shark</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>网卡设备选择</string>
    </property>
    <widget class="QComboBox" name="cb_netadaptor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>网卡1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>网卡2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>77</y>
      <width>651</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>流量分析</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>15</y>
       <width>641</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>128</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>IP版本</string>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>协议</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>源地址</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>目的地址</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>359</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>分析选项</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>源地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>45</y>
       <width>77</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>开始抓取</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>协议类型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_protocal">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>47</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UDP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ICMP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IGMP</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>目的地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>45</y>
       <width>77</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>一键导出</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_sourceIP">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>19</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_targetIP">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>19</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
